tests/integration/security/sds_ingress/ingress_test.go

```diff
@@ -37,6 +37,15 @@ func TestMain(m *testing.M) {
 		Run()
 }
 
+// TestSingleTlsGateway_SecretRotation tests a single TLS ingress gateway with SDS enabled.
+// Verifies behavior in these scenarios.
+// (1) create a kubernetes secret to provision server key/cert and client CA cert, and
```

Comment:
For TLS ingress gateway, client CA cert is not in the kubernetes secret. Please remove "and client CA cert" here and (2).

Reply:
Good catch!